src/csm_testing/tests/sat_functional/bootprep/test_bootprep.py

```diff
@@ -712,8 +726,11 @@ def test_session_template(self):
             self.items_to_delete['session_templates'].append(image['name'])
 
     def test_configs_images_and_session_templates(self):
-        """Test creating a bos session template"""
-        result = self.run_bootprep('configs-images-and-session-templates.yaml', '--format json')
+        """Test creating, skipping and overwriting configurations, images and session templates"""
+        bootprep_options = '--format json '
+        result = self.run_bootprep('configs-images-and-session-templates.yaml', bootprep_options)
+        skip_result = self.run_bootprep('configs-images-and-session-templates.yaml', bootprep_options)
+        overwrite_result = self.run_bootprep('configs-images-and-session-templates-overwrite.yaml', bootprep_options)
 
         report = json.loads(result.stdout.decode())
```

Comment:
We could check that the image reference in this first report has been deleted fully from IMS. Its ID will have changed, so this is possible to do. I believe we can check that `cray ims images describe` and `cray ims deleted images describe` both return a 404.